config/jobs/image-pushing/k8s-staging-infra-tools.yaml

```diff
@@ -8,6 +8,8 @@ postsubmits:
     decorate: true
     branches:
     - ^main$
+    # release tags
+    - ^v\d+.*
```

Comment:
this is a very loose regex and that should be fine, we have no reason to be pushing branches / tags with vN.... that are not release tags ...